include/cutlass/gemm/threadblock/threadblock_swizzle_streamk.h

```diff
@@ -43,10 +43,11 @@
 #include "cutlass/conv/conv3d_problem_size.h"
 #include "cutlass/gemm/threadblock/index_remat.h"
 
+#ifndef __CUDA_ARCH__
```

Comment:
can we use `#if !defined(__CUDACC_RTC__)` instead?

Reply:
Sure. Please find second commit.